drivers/vhm/vhm_ioreq.c

```diff
@@ -180,12 +180,6 @@ int acrn_ioreq_create_client(unsigned long vmid, ioreq_handler_t handler,
 			vmid);
 		return -EINVAL;
 	}
-	if (unlikely(vm->req_buf == NULL)) {
```

Comment:
But if this check is removed, where else is checked to assure that the vm->req is created?
Otherwise the buffer between kernel and low-level HV is not checked. This is not safe.
So I suggest that the check is still kept.